contracts/contracts/vault/OSonicVaultCore.sol

```diff
@@ -15,7 +15,7 @@ contract OSonicVaultCore is OETHVaultCore {
      * @notice Instant redeem is not supported on Sonic.
      * Use the asynchronous `requestWithdrawal` a `claimWithdrawal` instead.
      */
-    function redeem(uint256, uint256) external override {
+    function _redeem(uint256, uint256) internal override {
```

Comment:
Let's remove the virtual on the vault code that this inherited from.

Reply:
https://github.com/OriginProtocol/origin-dollar/pull/2433/commits/ba8a5961fa6c1077de54a0a983781bce2cda58f5